modular_skyrat/modules/milker/readme.md

```diff
@@ -0,0 +1,27 @@
+## Title: Expression
+
+MODULE ID: EXPRESSION
+
+### Description:
+
+Adds some milking capabilities.
+
+### TG Proc Changes:
+
+- N/A
+
+### Defines:
+
+- N/A
+
+### Master file additions
+
+- N/A
+
+### Included files that are not contained in this module:
+
+- N/A
+
+### Credits:
+
+Jake Park
```

Comment:
Uh... Are you sure Jake wrote this code? Lmao

Also, why are you making this a whole new module, rather than slotting it in with the already-existing Organic Interface code?

Reply:
Jake did it.

And, moved to the Organic Interface module.